socless/jinja.py

```diff
@@ -33,12 +45,33 @@ def maptostr(target_list):
         target_list (list): list containing python types
     Returns:
         List containing strings
+    Note:
+        May no longer be needed in Python3
     """
     return [str(each) for each in target_list]
 
 
+def vault(vault_id: str):
+    # A custom jinja Function which returns the content of a socless vault
+    # we expect it to be called as {vault( context.vault_id) }  and return the same value
+    # that current vault:vault-id would return
+    # Essentially a recreation of
+    return fetch_from_vault(vault_id, content_only=True)
+
+
+def fromjson(string_json: str) -> Any:
+    # This is a custom jinja Filter which expects stringified json and returns
+    # the output of calling json.loads on it.
+    return json.loads(string_json)
```

Comment:
What kind of error does this through if the use passes in something that isn't valid Stringified Json?
Can we wrap this in our own custom Exception that does a better job of explaining the cause of the failure? Or does Jinja override our exception anyways?

Reply:
updated this with a SoclessBootstrapError that specifies the error handling during `fromjson` jinja function:

```python
    try:
        return json.loads(string_json)
    except json.decoder.JSONDecodeError as e:
        raise SoclessBootstrapError(
            f"JSONDecodeError in `fromjson` Jinja filter/function. Error: {e} |\n String: {string_json}"
        )
```